peachjam_ml/models.py

```diff
@@ -147,6 +147,50 @@ def refresh_for_document(cls, document):
 
         return doc_embedding
 
+    @classmethod
+    def get_average_embedding(cls, pks):
+        """Get the average embedding for a set of documents."""
+        if not pks:
+            return None
+
+        if len(pks) == 1:
+            # if there's only one document, we can just return its embedding
+            doc_embedding = cls.objects.filter(document__pk=pks[0]).first()
+            if doc_embedding:
+                return doc_embedding.text_embedding
+
+        avg = (
+            DocumentEmbedding.objects.filter(document__in=pks)
+            .aggregate(avg=Avg("text_embedding"))
+            .get("avg")
+        )
+        if avg is not None and len(avg):
```

Comment:
This is the same as:

If avg:

Reply:
the value here is a numpy array, not a list. this is how we can check for truthiness for it

Reply:
ah of course